cpp/BUILD.bazel

```
@@ -172,6 +172,33 @@ cc_test(
     ],
 )
 
+cc_test(
+    name = "cluster_mode_base_user_api_test",
```

```suggestion
    name = "cluster_mode_api_test",
```
I suggest using a shorter name.